From 1480ef4ea9f71befbc22272c219b62ee5cd71d43 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 21 Jan 2022 15:17:37 -0800
Subject: [PATCH] Add sys/stat.h for S_IFDIR

../git/src/shared/mkdir-label.c:13:61: error: use of undeclared identifier 'S_IFDIR'
        r = mac_selinux_create_file_prepare_at(dirfd, path, S_IFDIR);

Upstream-Status: Backport [29b7114c5d9624002aa7c17748d960cd1e45362d]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/shared/mkdir-label.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/mkdir-label.c b/src/shared/mkdir-label.c
index e3afc2b666..f1df778966 100644
--- a/src/shared/mkdir-label.c
+++ b/src/shared/mkdir-label.c
@@ -7,6 +7,7 @@
 #include "selinux-util.h"
 #include "smack-util.h"
 #include "user-util.h"
+#include <sys/stat.h>
 
 int mkdirat_label(int dirfd, const char *path, mode_t mode) {
         int r;
-- 
2.39.2

